pkgs/development/compilers/llvm/5/libc++abi.nix

```diff
@@ -33,6 +33,7 @@ stdenv.mkDerivation {
     ''
     else ''
       install -d -m 755 $out/include $out/lib
+      install -m 644 lib/libc++abi.a $out/lib
```

Comment:
In glibc we have a static output. How about this one?

Reply:
I am not very supportive of static outputs. The issue is lots of things like pkgconfig hardcode the libdir. Instead, i would recommend adding an “enableStatic” argument that can be overriden.

Reply:
I modeled this on `libstdc++`, which just bundles both static and non-static outputs into the gcc derivation.

The static library is pretty small and we were building it anyway and throwing it away.  This is meant as a tiny patch that just allows us to build static libraries with `libcxxStdenv`, which was previously broken.  I tried not to make any architectural decisions :)

Reply:
The argument expected by `makeStaticLibraries` has a double negative — it's called `dontDisableStatic`.  If we're going to add an argument to things, shouldn't we stick to that convention?

```
[twey@uruz:~/nix/nixpkgs]$ grep -Irc enableStatic | awk -F: '{ s += $2 } END { print s }'
49

[twey@uruz:~/nix/nixpkgs]$ grep -Irc dontDisableStatic | awk -F: '{ s += $2 } END { print s }'
52
```

There doesn't seem to be consensus :laughing:

Reply:
But there are many more *files* with the double negative.

```
[twey@uruz:~/nix/nixpkgs]$ grep -Irl dontDisableStatic | wc -l
49

[twey@uruz:~/nix/nixpkgs]$ grep -Irl enableStatic | wc -l
19
```

Reply:
Yeah dontDisableStatic would be fine as well. The double negative is kind of awkward in my opinion though.

But yeah, this sounds okay if it's consistent with how we handle gcc.